src/vm/rejit.cpp

```diff
@@ -554,56 +630,24 @@ HRESULT ReJitManager::UpdateActiveILVersions(
             }
         }
 
-        CodeVersionManager * pCodeVersionManager = pModule->GetCodeVersionManager();
-        _ASSERTE(pCodeVersionManager != NULL);
-        CodeActivationBatch * pCodeActivationBatch = mgrToCodeActivationBatch.Lookup(pCodeVersionManager);
-        if (pCodeActivationBatch == NULL)
+        hr = UpdateActiveILVersion(&mgrToCodeActivationBatch, pModule, rgMethodDefs[i], fIsRevert, static_cast<COR_PRF_REJIT_FLAGS>(0));
```

Comment:
Rather than explicitly zeroing the flags, I suggest masking off the bits you don't want to flow. It is identical behavior now, but feels more intuitive about what I'd expect the behavior to be when a new flag gets added.

Reply:
I think this was actually a bug. I actually wanted to unconditionally set COR_PRF_REJIT_INLINING_CALLBACKS so the callback always was set for a method that had been passed to RequestReJIT, but instead was always clearing it.

I actually want to do `static_cast<COR_PRF_REJIT_FLAGS>(flags & COR_PRF_REJIT_INLINING_CALLBACKS)`